lunatrace/bsl/frontend/src/pages/project/builds/details/vulnerable-packages/VulnerablePackagesList.tsx

```diff
@@ -42,7 +43,7 @@ export const VulnerablePackagesList: React.FunctionComponent<FindingListProps> =
   // Todo: not sure if this should include below minimum severity packages or not, might be confusing. For now we are, though
   const pkgsToUpdate = vulnerablePackages.filter((pkg) => {
     if (pkg.trivially_updatable === 'yes' || pkg.trivially_updatable === 'partially') {
-      return true;
+      return isDirectDep(pkg); // only show the overall update queue for direct deps that could be updated, transitives don't work
```

Comment:
nit: comments should be on their own line. I usually find it more difficult to read a comment when i have to scan the line back and forth to keep both the referenced code and notes in my brain.

Reply:
Good point, forget you use a small screen by default